apps/frontend/static_src/scss/variables.scss

```
@@ -9,6 +9,7 @@ $body-bg: #f5f3f8;
 $font-size-root:             null !default;
 $font-size-base:             1rem !default; // Assumes the browser default, typically `16px`
 $font-size-sm:               $font-size-base * .875 !default;
+$font-family-base:           Kumbh Sans;
```

```suggestion
$font-family-base:           Kumbh Sans, Arial, Helvetica, sans-serif;
```

---

Here are Wagtail’s fallback fonts:

```suggestion
$font-family-base:           Kumbh Sans, -apple-system, BlinkMacSystemFont, "Segoe UI", system-ui, Roboto, "Helvetica Neue", Arial, sans-serif, Apple Color Emoji, "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji";
```

Link: https://github.com/wagtail/wagtail/blob/main/client/src/tokens/typography.js#L11